eos/references.yaml

```diff
@@ -1065,14 +1065,22 @@ F:2012A:
   inspire-id: Fleischer:2012fy
   title: On Branching Ratios of $B_s$ Decays and the Search for New Physics in $B^0_s\to
     \mu^+\mu^-$
+FK:2002A:
+  authors: Feldmann, T. and Kroll, P.
+  eprint:
+    archive: arXiv
+    id: oai:arXiv.org:1208.2843
+  inspire-id: Fleischer:2012fy
+  title: On Branching Ratios of $B_s$ Decays and the Search for New Physics in $B^0_s\to
+    \mu^+\mu^-$
 FFKMvD:2013A:
   authors: Faller, Sven and Feldmann, Thorsten and Khodjamirian, Alexander and Mannel,
     Thomas and van Dyk, Danny
   eprint:
     archive: arXiv
-    id: oai:arXiv.org:1310.6660
-  inspire-id: Faller:2013dwa
-  title: Disentangling the Decay Observables in $B^- \to \pi^+\pi^-\ell^-\bar\nu_\ell$
+    id: oai:arXiv.org:hep-ph/0201044
+  inspire-id: Feldmann:2002kz
+  title: Mixing of pseudoscalar mesons
```

Comment:
Why are you replacing the info for the 2013 paper with a different one?

Reply:
Because I messed up!